_im-plugin/index-transforms/transforms-apis.md

```diff
@@ -291,6 +293,7 @@ GET _plugins/_transform/<transform_id>
   "transform": {
     "transform_id": "sample",
     "schema_version": 7,
+    "continuous": true,
```

Comment:
There is another response just below this for when you "get details of all transform jobs by omitting transform_id", could you add this line to that response as well, for consistency?

Reply:
That is an oops from me. Added back in.